x-pack/legacy/plugins/uptime/server/lib/domains/__tests__/license.test.ts

```diff
@@ -8,31 +8,31 @@ import { ILicense } from '../../../../../../../plugins/licensing/server';
 import { licenseCheck } from '../license';
 
 describe('license check', () => {
-  let mockLicense: Pick<ILicense, 'isActive' | 'isOneOf'>;
+  let mockLicense: Pick<ILicense, 'isActive' | 'hasAtLeast'>;
```

Comment:
@justinkambic looks like we should update CODEOWNERS to make uptime owner of the plugin